raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -292,21 +311,58 @@ def sanity_check(state: MediatorTransferState) -> None:
         assert state.secrethash == first_pair.payer_transfer.lock.secrethash
 
     for pair in state.transfers_pair:
-        assert is_send_transfer_almost_equal(pair.payee_transfer, pair.payer_transfer)
+        payee_channel = get_payee_channel(
+            channelidentifiers_to_channels=channelidentifiers_to_channels,
+            transfer_pair=pair,
+        )
+
+        # Channel could have been removed
```

Comment:
hummm, this is wrong IMO. The transfer should be over before the channel can be settled, if that is not the case we probably have a bug.